dev-libs/rapidjson/rapidjson-1.1.0.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A fast JSON parser and generator for C++ with both SAX/DOM style API"
+DESCRIPTION="RapidJSON is a JSON parser and generator for C++. \
+It was inspired by RapidXml. RapidJSON is small but complete. \
+RapidJSON is fast. Its performance can be comparable to strlen().\
+RapidJSON is self-contained and header-only. RapidJSON is memory-friendly. \
+RapidJSON is Unicode-friendly"
+HOMEPAGE="https://github.com/Tencent/rapidjson"
+COPYRIGHT="2015 THL A29 Limited, a Tencent company
+		2015 Milo Yip"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/Tencent/rapidjson/archive/v1.1.0.tar.gz"
+CHECKSUM_SHA256="bf7ced29704a1e696fbccf2a2b4ea068e7774fa37f6d7dd4039d0787f8bed98e"
+SOURCE_FILENAME="rapidjson-1.1.0.tar.gz"
+SOURCE_DIR="rapidjson-1.1.0"
+PATCHES="rapidjson-1.1.0.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	rapidjson$secondaryArchSuffix = $portVersion
+	lib:rapidjson$secondaryArchSuffix = $portVersion compat >= $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	rapidjson${secondaryArchSuffix}_devel = $portVersion compat >= $portVersion
+	devel:rapidjson$secondaryArchSuffix = $portVersion compat >= $portVersion
+	"
+REQUIRES_devel="
+	rapidjson$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:c++$secondaryArchSuffix
+	cmd:cmake
+	cmd:make
+	"
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+	cmake ..
+	make
+}
+
+INSTALL()
+{
+	mkdir -p $libDir/pkgconfig
+	mkdir -p $libDir/cmake/RapidJSON
+	mkdir -p $includeDir/rapidjson
+
+	cp -r $sourceDir/include/rapidjson/* $includeDir/rapidjson/
+	cp $sourceDir/build/RapidJSON.pc $libDir/pkgconfig
+	cp $sourceDir/build/RapidJSONConfig* $libDir/cmake/RapidJSON
+
+	packageEntries devel $developDir $libDir
+}
```

Comment:
The last line is missing the famous LF (linefeed) character, "ascii" code 0x0A.
To add it, you can use vi:
```
vi rapidjson-1.1.0.recipe
:w
:q
```
Could you open the vile with vi and type: